resources/views/dashboard/admin/events/edit.blade.php

```diff
@@ -56,8 +56,10 @@
             {!! Form::textArea('description', $event->description, ['class' => 'form-control text-editor']) !!}
         </div>
         <div class="form-group">
-            {!! Form::label('banner', 'Upload Banner') !!}
+            {!! Form::label('banner', 'Upload Banner or Enter Banner URL (Optional)') !!} 
```

Comment:
Maybe let's split this into 2 labels? I can see this wording being confusing because it's one or the other and this makes it seem like both are optional

Or maybe just remove `(Optional)` because I think having the OR between the inputs is clear enough